README.md

```diff
@@ -1,39 +1,89 @@
 <a href="https://nimblebox.ai/" target="_blank"><img src="./assets/built_at_nbx.svg" align="right"></a>
+[![PyPI - Python
+Version](https://img.shields.io/badge/python-3.6%20%7C%203.7%20%7C%203.8%20%7C%203.9-blue)](https://pypi.org/project/nbox/)
+[![Downloads](https://pepy.tech/badge/nbox)](https://pepy.tech/project/nbox)
+![GitHub](https://img.shields.io/badge/license-Apache--2.0-blueviolet)
 
-# 🏖️ Nbox
+## 🧐 What is Nbox?
 
-`nbox` is NimbleBox.ai's official SDK.
+`nbox` provides first class support API for all NimbleBox.ai infrastructure (NBX-Build, Jobs, Deploy) and services (NBX-Workspaces) components. Write jobs using `nbox.Operators`
 
-> The entire purpose of this package is to make using ML 🥶.
+# 🤷Why nimblebox
 
-```
-pip install nbox
-```
+- Write and execute code in Python
```

Comment:
Make these catchy and quick.
`import notebooks from local machine`

Reply:
we can actually remove this part because I have added more catchy things below in features